client_code/portable/__init__.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2021 anvilistas
+import json as _json
+from functools import wraps as _wraps
+
+import anvil.server as _server
+
+from ._register import register
+from ._serialize import UNHANDLED as _U
+from ._serialize import reconstruct, serialize
+
+__version__ = "0.0.1"
+
+
+def _dumps(obj):
+    s = serialize(obj)
+    u = s.pop(_U)
+    return _json.dumps(s), u
+
```

Comment:
can we have some more meaningful variable names here? I have no idea what 's', '_U' and 'u' mean.

Reply:
absolutely!